coil-base/src/main/java/coil/memory/MemoryCache.kt

```diff
@@ -12,7 +11,6 @@ import kotlinx.android.parcel.Parcelize
 /**
  * An in-memory cache of recently loaded images.
  */
-@ExperimentalCoilApi
```

Comment:
Removing the experimental status on `MemoryCache` so `placeholderKey` doesn't have to be experimental.